### ISSUE TITLE
Error in graphics layout while performing GWAS using GAPIT

### ISSUE THREAD

**Issue by vikram207** [jiabowang/GAPIT#111]
I have been trying to run GWAS using GAPIT and I keep getting this error for past few days:

**[1] "--------------------- Welcome to GAPIT ----------------------------"
[1] "Blink"
[1] "--------------------Processing traits----------------------------------"
[1] "Phenotype provided!"
[1] "The 1 model in all."
[1] "BLINK"
[1] "GAPIT.DP in process..."
[1] "Converting genotype..."
[1] "Converting HapMap format to numerical under model of Middle"
[1] "Perform numericalization"
[1] "Succesfuly finished converting HapMap which has bits of 2"
[1] "Converting genotype done."
[1] "GAPIT will filter marker with MAF setting !!"
[1] "The markers will be filtered by SNP.MAF: 0"
maf_index
 TRUE 
21033 
[1] "Calculating kinship..."
[1] "Number of individuals and SNPs are  587  and  21033"
[1] "Kinship created!"
[1] "Calling prcomp..."
[1] "Creating PCA graphs..."
[1] "Joining taxa..."
[1] "Exporting PCs..."
[1] "PC created"
[1] "Filting marker for GAPIT.Genotype.View function ..."
Error in graphics::layout(p, ...) : 
  argument "p" is missing, with no default
In addition: There were 50 or more warnings (use warnings() to see the first 50)**

I thought the error was due to problems with my data. But tutorial data leads to the same error. Can somebody provide a solution to this error?

**Comment by jiabowang** [comment]
I have not seen this error with demo data. Please see whether the pdf files were open when the GAPIT ran. and update your R and packages.

**Comment by zhen0506** [comment]
I am getting the same error. Did you figure out the reason?

**Comment by guorongdi** [comment]
I am getting the same error. Did you figure out the reason

**Comment by zhen0506** [comment]
Yes. Gapit does not allow tibble (I guess you used tidyverse.). The phenotype or genotype data need to be changed to default dataframe using as.data.frame() 
Hope this works for you as well.

**Comment by guorongdi** [comment]
这是一条自动回复的消息，已收到你的邮件。
